x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecations/deprecations_list.test.ts

```diff
@@ -63,16 +63,16 @@ describe('ES deprecations table', () => {
     const reindexDeprecation = esDeprecationsMockResponse.deprecations[3];
 
     // Since upgradeStatusMockResponse includes ML and reindex actions (which require fetching status), there will be 3 requests made
```

Comment:
This comment seems out of date since it refers to 3 requests, but the code now expects 4 more requests. Can we update it?